arm/Microsoft.Sql/managedInstances/deploy.bicep

```
@@ -60,10 +60,10 @@ param proxyOverride string = 'Proxy'
 @description('Optional. Whether or not the public data endpoint is enabled.')
 param publicDataEndpointEnabled bool = false
 
-@description('Optional. Id of the timezone. Allowed values are timezones supported by Windows.')
+@description('Optional. ID of the timezone. Allowed values are timezones supported by Windows.')
 param timezoneId string = 'UTC'
 
-@description('Optional. The Id of the instance pool this managed server belongs to.')
+@description('Optional. The resource ID of the instance pool this managed server belongs to.')
 param instancePoolId string = ''
```

Guess this should also reference ResourceID in the actual parameter? This needs an update to the Parameter file too.

```suggestion
param instancePoolResourceId string = ''
```